tests/extensions/snippets/snippets (block).txt

```diff
@@ -8,18 +8,3 @@ d.txt
 ---8<--- 
 d.txt
 ---8<--- 
-
```

Comment:
Why are we removing tests? Or I should say moving them?

Reply:
The comment "# Nested inline won't work" was not a valid path, which made the tests fail before adding the option, splitting was a way to resolve this (putting the comment outside the snippet block). I'll re-merge them.

Reply:
Okay, that is good. For unittests, you'll probably want to add a test to `test_targeted.py`.  Most likely you'd call `markdown` with the `snippets` extension and content with a bad file and capture the `FileNotFoundError`.

Reply:
Done